src/clj/rems/middleware.clj

```diff
@@ -63,8 +63,8 @@
   [handler]
   (fn [request]
     (binding [context/*user* (keywordize-keys (:identity request))]
-      (when context/*user*
-        (users/add-user-raw! (get-user-id) context/*user*))
+      ;; (when context/*user*
+      ;;   (users/add-user-raw! (get-user-id) context/*user*))
```

Comment:
this probably should be removed now that user is created on actual login and not on every request?

Reply:
Yeah, I think so.